virt/virtual_machines/virt-create-vms.adoc

```diff
@@ -39,12 +39,15 @@ include::modules/virt-creating-vm.adoc[leveloffset=+1]
 :virtualmachine!:
 
 include::modules/virt-vm-storage-volume-types.adoc[leveloffset=+1]
+include::modules/virt-about-runstrategies-vms.adoc[leveloffset=+1]
+* See xref:../../machine_management/deploying-machine-health-checks.adoc#machine-health-checks-about_deploying-machine-health-checks[Deploying machine health checks] for further details on deploying and enabling machine health checks.
```

Comment:
This looks a little weird with how it's just jammed at the end of the module.

Suggest moving these bullets to the below "Additional resources" section (and putting the existing item in bullets there, which it already should have been).